recipes/ux.rb

```diff
@@ -8,12 +8,19 @@
 #
 
 log "Create jellyfish user"
-
 user "jellyfish" do
   comment "jellyfish user"
   shell "/bin/bash"
 end
 
+log "Create jellyfish sudoers file"
+file "/etc/sudoers.d/jellyfish" do
```

Comment:
You'll need to work this out.  This will introduce a big vulnerability.  I'm going to close the PR, please re-submit once you have it working.